components/mpas-albany-landice/driver/glc_comp_mct.F

```diff
@@ -1487,26 +1491,24 @@ subroutine glc_export_mct(g2x_g, errorCode)
 
          ! Get variables from pools
          call mpas_pool_get_subpool(block % structs, 'geometry', geometryPool)
-         call mpas_pool_get_subpool(block % structs, 'thermal',  thermalPool)
+         call mpas_pool_get_subpool(block % structs, 'thermal', thermalPool)
+         call mpas_pool_get_subpool(block % structs, 'timeAveraging', timeAveragingPool)
 
          call mpas_pool_get_array(geometryPool, 'upperSurface', upperSurface)
          call mpas_pool_get_array(meshPool, 'layerThicknessFractions', layerThicknessFractions)
          call mpas_pool_get_array(geometryPool, 'thickness', Thickness, timeLevel = 1)
          call mpas_pool_get_array(thermalPool, 'temperature', temperature)
+         ! Should the next field be the full field or just the subset that is "applied" to the ice covered extent?
```

Comment:
We'll need to decide what to do about this question.  We should somehow account for the mass imbalance.  @jonbob , do you have suggestions about how we handle the situation where ELM sends ice ablation to MALI but MALI does not have enough ice in a grid cell to satisfy it?

Reply:
Noting that we've removed support for bareIceAblation and are only calculating / using bareIceAblationApplied now. But we can still use the full and applied versions of sfcMassBal field to track this if necessary.